dive/consts.py

```diff
@@ -1,2 +1,52 @@
 MAX_FILE_SIZE_MB = 10  # 10 MB limit
 MAX_FILE_SIZE_BYTES = MAX_FILE_SIZE_MB * 1024 * 1024
+
+HAEADERS = [
+    {
+        "key": "1",
+        "label": "One",
+    },
+    {
+        "key": "2",
+        "label": "Two",
+    },
+    {
+        "key": "3",
+        "label": "Three",
+    },
+    {
+        "key": "4",
+        "label": "Four",
+    },
+    {
+        "key": "5",
+        "label": "Five",
+    },
+]
+
+LANGUAGES = [
```

Comment:
I am not sure if there are standard keys available for languages, just like iso and iso3 for countries. Good to know about them and if they exist use the standard. Not a must to change but something to think about keeping things as standard as possible.